e2e/cypress/integration/channel/message_permalink.js

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/* eslint max-nested-callbacks: ["error", 7] */
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+import users from '../../fixtures/users.json';
+
+const cypressConfig = require('../../../cypress.json');
+
+describe('Message permalink', () => {
+    before(() => {
+        // # Login and go to /
+        cy.apiLogin('user-1');
+        cy.visit('/');
+    });
+
+    it('M13675-Copy a permalink and paste into another channel', () => {
+        const message = 'Hello' + Date.now();
+        const channelName = 'test-message-channel-1';
+
+        // # Create new DM channel with user's email
+        cy.apiGetUsers(['user-1', 'sysadmin']).then((userResponse) => {
+            const userEmailArray = [userResponse.body[1].id, userResponse.body[0].id];
+
+            cy.apiCreateDirectChannel(userEmailArray).then((response) => {
+                const direcrChannel = response.body;
+                cy.getCurrentTeamName().then((teamName) => {
+                    var currentTeamName = `${teamName}`.toLowerCase();
+                    const url = `${cypressConfig.baseUrl}/${currentTeamName}/messages/@${users.sysadmin.username}`;
+                    cy.visit(url);
```

Comment:
This line is failing since URL format includes team name and not its display name.  I believe you may remove this ``cy.visit(url)`` since the next line actually clicks the channel.